crates/ruff_python_formatter/src/context.rs

```diff
@@ -210,3 +220,109 @@ where
             .set_node_level(self.saved_level);
     }
 }
+
+/// The current indent level of the formatter.
+///
+/// One can determine the the width of the indent itself (in number of ASCII
+/// space characters) by multiplying the indent level by the configured indent
+/// width.
```

Comment:
The "number of ASCII space characters" is, I guess, a little less clear when talking about tabs. Like, to determine the number of tabs, it'd be the number of ASCII space characters divided by the tab width? But IIRC we don't support tabs within docstrings anyway, so perhaps moot here.

Reply:
> But IIRC we don't support tabs within docstrings anyway,

Yeah that's exactly why I used the naming I did. I felt like if I used more generic naming, it might be too easy to accidentally use this method in an inappropriate way.